### PR TITLE
 build 0.0.6

### PR DESCRIPTION
**Pull request by boldorider4**
streamlit-plotly-events 0.0.6 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-3586] 
- [Upstream repository](https://github.com/ethanhe42/streamlit-plotly-events/tree/0.0.5)

[PKG-3586]: https://anaconda.atlassian.net/browse/PKG-3586?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Review by skupr-anaconda — APPROVED** [review]

File: abs.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+upload_channels:
+  - sfe1ed40
+
+channels:
+  - https://staging.continuum.io/prefect/fs/pydeck-feedstock/pr4/6f4c117
```
Comment (JeanChristopheMorinPerso) [inline abs.yaml:5]:
```suggestion
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -35,8 +35,14 @@ test:
 about:
   home: https://github.com/null-jones/streamlit-plotly-events
   summary: Plotly chart component for Streamlit that also allows for events to bubble back up to Streamlit.
+  description: |
+    Plotly chart component for Streamlit that also allows for events to bubble back up to Streamlit.
+    Ever wanted to harness those awesome events from Plotly charts inside of Streamlit? So did I, so now you can!
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:40]:
```suggestion
```
This doesn't add any useful information and is more marketing content. (I know it comes from upstream, but we don't need to add this here).

Comment (boldorider4) [reply recipe/meta.yaml:40]: sorry, sometimes copy paste takes it too far.

**Pull request merged**